GoodDollar/GoodDAPP
# (Chore) [RN] 1199 Re-integrate hot reloader for web

# Description

Hot reloading for web was not working on react-native branch, this PR fixes that.

About #1199 


# How Has This Been Tested?

1. Run web and test signup router by making changes in code and let hot reloading do it's job
2. Run web and test logged in router by making changes in code and let hot reloading do it's job
3. Run native and no differences should be found

# Checklist:
